WORKSPACE

```diff
@@ -286,3 +288,11 @@ maven_jar(
   name = "org_mockito_mockito_all",
   artifact = "org.mockito:mockito-all:1.10.19",
 )
+
+git_repository(
+  name = "io_bazel_rules_pex",
+  remote = "https://github.com/benley/bazel_rules_pex.git",
```

Comment:
Agreed pypi would be awkward, but maven does have support for tar.gz artifacts. See http://stackoverflow.com/questions/32088885/install-tar-gz-file-in-maven-repository